src/calculus/HCAT.jl

```diff
@@ -37,23 +37,22 @@ julia> H*(ones(3),ones(3))
 
 """
 struct HCAT{M, # number of codomains  
-	    N, # number of AbstractOperator 
-	    L <: NTuple{N,AbstractOperator},
-	    P <: NTuple{N,Union{Int,Tuple}},
-	    C <: Union{NTuple{M,AbstractArray}, AbstractArray},
-	    } <: AbstractOperator
+            N, # number of AbstractOperator 
+            L <: NTuple{N,AbstractOperator},
+            P <: NTuple{N,Union{Int,Tuple}},
+            C <: Union{NTuple{M,AbstractArray}, AbstractArray},
```

Comment:
I think this one could be simplified? In the idea of moving to ArrayPartition to represent product spaces, here the arrival space can just be an AbstractArray, since ArrayPartition <: AbstractArray. Or am I missing something?

Reply:
yes I guess it can be simplified, was just lazy! 😄